### PR TITLE
Improvement/local-installability

### PR DESCRIPTION
**Pull request by cun3yt**
@ncardeli 
In order to solve local install-ability for better portability I made some changes. Details of the problem are in [this issue](https://github.com/ncardeli/page-computed-style/issues/2).

I know the package is intended for global installation but better we convert it to be able to run both globally and locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,16 @@ For help and more options, type:
 page-computed-style -h
 ```
 
+## Installing & Using Locally
```
Comment (ncardeli) [inline README.md]: I'd like to keep the readme as small and focused as possible. I expect most people will install the package globally, so I wouldn't add instructions for installing and using locally in the project readme. Instead we could create a separate document file or add an entry to the wiki. What do you think?

File: README.md [hunk 2]
```diff
@@ -49,6 +49,16 @@ For help and more options, type:
...
+
+If you would rather installing the package locally, omit `-g` parameter and the usage will be directly from `node_modules/page-computed-style/index.js`:
```
Comment (ncardeli) [inline README.md]: I'm not a native english speaker, but I think it should be "If you would rather install" instead of "If you would rather installing"

**Review by ncardeli — CHANGES_REQUESTED** [review]
Great contribution! Thanks a lot. I've left some comments regarding the readme file changes that should be addressed before merging.

File: README.md (hunk 2 above)
Comment (cun3yt) [reply README.md]: Neither am I. You are right, it should be "... rather install".

File: README.md (hunk 1 above)
Comment (cun3yt) [reply README.md]: 👍

Comment (cun3yt) [reply README.md]: A separate file will be easier to manage than a wiki entry I believe. I will update accordingly. Thanks

**Comment by cun3yt** [comment]
@ncardeli I moved the section to a separate file under the newly created folder`docs`. The new file has some updates, too.

**Review by ncardeli — APPROVED** [review]

**Pull request merged**